docs/language/accounts.mdx

```diff
@@ -220,6 +220,14 @@ pub struct AuthAccount {
     /// The path must be a storage path, i.e., only the domain `storage` is allowed
     pub fun borrow<T: &Any>(from: StoragePath): T?
 
+    /// Returns true if the object in account storage under the given path satisfies the given type, 
+    /// i.e. could be borrowed using the given type.
+    ///
+    /// The given type must not necessarily be exactly the same as the type of the borrowed object.
+    ///
+    /// The path must be a storage path, i.e., only the domain `storage` is allowed.
+    pub fun check<T: &Any>(from: StoragePath): Bool
+
```

Comment:
@darkdrag00nv2 The Cadence documentation us currently being migrated to the https://github.com/onflow/docs repository. Could you please open a PR there, on top of https://github.com/onflow/docs/pull/58 (if it is still open at the time), then remove the docs change from this PR (another PR is removing the documentation)? Sorry for the inconvenience, I'm happy to assist

Reply:
Sure, I'll move it.

Reply:
Moved the docs changes to https://github.com/onflow/docs/pull/68 and removed from this PR.

Reply:
Thank you, and sorry about the confusion there